sdk/core/azure-core-tracing-opentelemetry/test/ut/azure_core_test.cpp

```diff
@@ -10,6 +11,26 @@ int main(int argc, char** argv)
   // End users need to decide if SIGPIPE should be ignored or not.
   signal(SIGPIPE, SIG_IGN);
 #endif
+  // Declare a signal handler to report unhandled exceptions on Windows - this is not needed for
+  // other
+// OS's as they will print the exception to stderr in their terminate() function.
```

Comment:
```suggestion
  // OS's as they will print the exception to stderr in their terminate() function.
```

Reply:
Code is no longer there, so this change is unnecessary.

Reply:
Looks like the suggestion to indent the comment still applies.
See this comment:
https://github.com/Azure/azure-sdk-for-cpp/pull/4015#discussion_r1002037939